app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = " "
+    base_title =  t ".home"
+    page_title.empty? ? base_title : page_title + " | " + base_title
```

Comment:
1. Anh muôn sử dụng check blank? để thay thế empty? chỗ này được k em ?
2. Và 2 method này khác nhau gì em ?

Reply:
Em nghĩ trong trường hợp này là được vì mục đích ở đây xét việc hiện thị title nhưng sẽ không được khi vẫn khai báo title nhưng để toàn khoảng trắng trong tên title.

Em nghĩ là sự khác nhau của 2 method này là:
+Blank? sẽ trả về giá trị true khi ta gõ " ".blank? vậy có nghĩa là blank sẽ trả về giá trị true khi mà  gặp 1 chuỗi toàn khoảng trắng hay 1 object false hoặc rỗng.
+Empty? sẽ trả về giá trị false khi ta gõ " ".empty? có nghĩa là empty? sẽ trả giá trị true khi mà độ dài của chuỗi cũng như mảng bằng 0